src/Components/LocationForm.js

```diff
@@ -18,32 +16,34 @@ export default function LocationForm({ onAddLocations }) {
   const [isError, setIsError] = useState(false);
   const [status, setStatus] = useState(null);
 
-  /* console.log('added location', location); */
+  console.log('added location', location);
```

Comment:
please consider removing this console.log

Reply:
Thank you. I'm gonna delete it.